server/utils/stripe.js

```diff
@@ -0,0 +1,70 @@
+const stripe = require('stripe')(process.env.STRIPE_SECERT);
+const Booking = require('../models/Booking');
+const Sitter = require('../models/Sitter');
+
+const booking = Booking.findById(req.params.id);
+exports.createStripeCustomer = async () => {
+  const user = await User.findById(booking.userId);
+  const { name, email } = user;
+  const customer = await stripe.create({
+    name: name,
+    email: email,
+  });
+  return customer;
+};
+
+exports.chargeCustomer = async (id) => {
+  const sitter = await Sitter.findById(booking.sitterId);
+  const startDate = new Date(booking.start);
+  const endDate = new Date(booking.end);
+  const diff = Math.abs(startDate - endDate);
+  const hours = diff / 36e5;
+  const amount = (hours * sitter.rate).toFixed(2);
+  const processingFee = 5;
+  const CURRENCY = 'usd';
+  stripe.charges.create({
+    amount: amount + processingFee,
+    currency: CURRENCY,
+    customer: id,
+  });
+};
+
+exports.createProduct = async () => {
+  const PRODUCT_NAME = 'Monthly Subscription';
+  const PRODUCT_TYPE = 'service';
```

Comment:
So - I don't think we'll do monthly subscriptions in this app. Or would we create pet sitters products at all

Reply:
in the ticket there is a link to subscriptions that's why I created these methods
but I will take them off